specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -4934,7 +4939,7 @@
           "type": "string",
           "description": "Resource type."
         },
-        "etag": {
+        "eTag": {
```

Comment:
from ARM common definition. looks correct should be etag.  and which item in ADO is what you are trying to fix?

Reply:
The issue reported is this

Additional properties not allowed: eTag
Json path: $.value[*].eTag

The property is there in the model. But the error is still showing. So, am suspecting that either swagger is case sensitive or expecting camel casing on the property. That's the fix am trying to make.

Reply:
in this case, I will recommend you to fix it in your service side. BTW, how about other correctness in ADO items? will you also fix?

Reply:
Wondering why this would be a fix on the service side. The response from the service is correct. The casing needs to be updated on the swagger specs. Hence this fix.

For the other issues in the ADO item - Not all the APIs mentioned in that work item is owned by my team. There are multiple teams within cost management that own different APIs. Each team will will own the fixes for their APIs.

Reply:
per ARM guide: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md
You can find etag in the document. it should be etag, not eTag.

Reply:
Thank you! Reverted the change casing update for etag. Are we good with the other change?

Reply:
LGTM,